package.json

```
@@ -79,6 +79,11 @@
         "command": "cairo.viewAnalyzedCrates",
         "title": "View currently analyzed crates",
         "category": "Cairo"
+      },
+      {
+        "command": "cairo.showSyntaxTree",
+        "title": "Show syntax tree of the current file content",
+        "category": "Cairo"
```

```suggestion
        "command": "cairo.viewSyntaxTree",
        "title": "View syntax tree of the current file content",
        "category": "Cairo"
```

to be consistent with `viewAnalyzedCrates`

---

It was on purpose to enable easier searching via cmd+shift+P - easier to type view to see analyzed crates and show to see syntax tree

---

I don't think this approach would work anyway when we happen to add more commands of this style, wdyt?

---

Ye, whatever tbh